sdks/java/extensions/euphoria/euphoria-core/src/main/java/org/apache/beam/sdk/transforms/windowing/WindowDesc.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.transforms.windowing;
+
+import org.apache.beam.sdk.values.WindowingStrategy;
+
+/**
+ * Exposes window package private properties for testing
+ *
+ * @param <T> type of input element
+ */
+public class WindowDesc<T> {
```

Comment:
no we, need this in composite operators. Anyway, I think we should remove triggeredBy and accumulation mode steps from builders and replace it with single windowBy step. This would also allow us to remove this class.

Reply:
moved in https://github.com/seznam/beam/pull/36